__tests__/app_test_.js

```diff
@@ -0,0 +1,89 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+require("jest-sorted");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("App tests are running", () => {
+  test("app.tests is running...", () => {});
+});
+
+describe("GET /api/categories", () => {
+  test("200: Responds with an array of categories objects with each having the slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
+        expect(categories).toHaveLength(4);
+        categories.forEach((categories) => {
+          expect(categories).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+});
+
+describe("GET Error Handling", () => {
+  test("404: Responds with a correct error message for an invalid get request path", () => {
+    return request(app)
+      .get("/api/notfound")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toEqual("Not Found");
+      });
+  });
+});
+
+describe("GET /api/reviews/:review_id", () => {
+  test("200: Responds with an objects containing (review_id, title, review_body, designer, review_img_url, votes(with a defalut of 0), category , owner, created_at defaults to the current timestam", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toEqual(
```

Comment:
i would have this as review rather than reviews, you should only be expecting one review here! you'll have to change it in your controller too just for clarity